frontend/src/components/admin/AdminPostingCard.tsx

```diff
@@ -125,7 +125,7 @@ const AdminPostingCard = ({
               <Icon as={BsPeopleFill} w={6} pr={2} />
               {status === PostingFilterStatus.DRAFT
                 ? "Not accepting registrants"
-                : `${numVolunteers} volunteers`}
+                : `${numVolunteers} volunteer${numVolunteers > 1 ? "s" : ""}`}
```

Comment:
```suggestion
                : `${numVolunteers} volunteer${numVolunteers != 1 ? "s" : ""}`}
```
edge case for num = 0 (0 volunteers)